docs/python_docs/themes/mx-theme/mxtheme/footer.html

```diff
@@ -4,13 +4,12 @@
             <div class="col-4">
                 <h4 class="footer-category-title">Resources</h4>
                 <ul class="contact-list">
-                    <li><a class="u-email" href="mailto:dev@mxnet.apache.org">Dev list</a></li>
-                    <li><a class="u-email" href="mailto:user@mxnet.apache.org">User mailing list</a></li>
-                    <li><a href="https://cwiki.apache.org/confluence/display/MXNET/Apache+MXNet+Home">Developer Wiki</a></li>
-                    <li><a href="https://issues.apache.org/jira/projects/MXNET/issues">Jira Tracker</a></li>
-                    <li><a href="https://github.com/apache/incubator-mxnet/labels/Roadmap">Github Roadmap</a></li>
+                    <li><a href="https://lists.apache.org/list.html?dev@mxnet.apache.org">Dev list</a> (<a class="u-email" href="mailto:dev-subscribe@mxnet.apache.org">subscribe</a>)</li>

```

Comment:
This footer link here is different from main site. 
And the parentheses appear black while "subscribe" text is white. We can put parentheses inside the inner `<a/>` tag like this `<a class="u-email" href="mailto:dev-subscribe@mxnet.apache.org">(subscribe)</a>`

Reply:
thanks for catching it. I wasn't able to use the same reference link in python site as I did in static site, and thus made the compromise of linking to dev@ list. is there a solution with which I can link it to the same page as static site?

Reply:
I can only think of hardcoding the link, but I think this work around is ok